core/app/layout/layout.go

```diff
@@ -124,6 +131,14 @@ var binABIToDir = map[string]string{
 	"x86":         "android-x86",
 }
 
+var DirToBinABI = func() map[string]string {
```

Comment:
It's more idiomatic to declare a var and initialize it with an `init()` function.
That said, why not offer it as a function that takes a dir parameter and returns the bin ABI?
Either way it a new public symbol - please document it.